Bindings/Java/tests/CMakeLists.txt

```diff
@@ -86,6 +86,14 @@ macro(OpenSimAddJavaTest TESTNAME)
                  DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/TestVisualization")
     file(COPY "${OPENSIM_SHARED_TEST_FILES_DIR}/gait10dof18musc_subject01.osim"
                  DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/TestEditProperties")
+    file(COPY "${OPENSIM_SHARED_TEST_FILES_DIR}/gait2354_simbody.osim"
```

Comment:
Is it possible to use model/setup files that are already in the repo (e.g., the files used by testScale.cpp), particularly since this isn't a correctness/regression test? There are already 4 files named "gait2354_simbody.osim" in the repo.

Reply:
Thanks for the note @tkuchida I'm actually using the recommended method of having these files added to the shared testing folder, we should make sure other tests are using it from there. This may or maynot be part of this PR but good point.

Reply:
Yes, this is the recommended method, but ideally, one of the pre-existing gait2354 models would have been moved to the shared folder.

Reply:
Added a comment to this effect in Issue #206.

Reply:
Thanks @tkuchida 👏

Reply:
While we want to move to shared resources to avoid duplication, I think copying essentially the same files in testScale violates that spirit. I would prefer we use the testScale files that already exist instead of making yet another copy.